orbitdeterminator/propagation/sgp4_prop_string.py

```diff
@@ -1,3 +1,5 @@
+"""SGP4 propagator. This is a wrapper around the PyPI SGP4 propagator."""
```

Comment:
Maybe here write why this script is different thatn sgp4_prop. Cause I see its the same comment @aryadas98